service/src/test/resources/application.yml

```diff
@@ -0,0 +1,14 @@
+server:
+  port: ${GP2GP_SERVER_PORT:8080}
+
+management:
+  endpoints:
+    web:
+      base-path: /
+      path-mapping.health: healthcheck
+      exposure:
+        include: info, health, metrics, mappings
+storage:
```

Comment:
This these should be gp2gp.storage? Would avoid any potential naming conflicts if there is some other spring module out there using the "storage" root config property